okhttp/src/main/java-templates/okhttp3/internal/Version.kt

```diff
@@ -13,13 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package okhttp3.internal;
+package okhttp3.internal
 
-public final class Version {
-  public static String userAgent() {
-    return "okhttp/${projectVersion}";
-  }
-
-  private Version() {
-  }
+object Version {
+  @JvmStatic
+  val userAgent = "okhttp/${projectVersion}"
```

Comment:
Make top level, remove `object`, and add

```kotlin
@file:JvmName("Version")
```

To avoid the needless singleton instance.

Reply:
Oh Kotlin, some day I will understand you. Fixed.